src/geometry/euclidean.lean

```diff
@@ -30,15 +30,6 @@ theorems that need it.
 
 -/
 
-/-- A `euclidean_affine_space V P` is an affine space with points `P`
-over an `inner_product_space V`. -/
-abbreviation euclidean_affine_space (V : Type*) (P : Type*) [inner_product_space V]
-    [metric_space P] :=
-normed_add_torsor V P
-
-example (n : Type*) [fintype n] : euclidean_affine_space (euclidean_space n) (euclidean_space n) :=
-by apply_instance
-
```

Comment:
Will need an update to the module doc string, to discuss how `[inner_product_space V] [metric_space P] [normed_add_torsor V P]` is the way to declare a Euclidean affine space, and why it's done that way.